src/status_im/contexts/wallet/swap/setup_swap/view.cljs

```diff
@@ -303,6 +312,19 @@
                       (when refetch-interval
                         (js/clearInterval refetch-interval)
                         (set-refetch-interval nil))))
+    (rn/use-effect on-asset-to-pay-change [asset-to-pay])
+    (rn/use-effect
+     (fn []
+       (when (and swap-amount refetch-interval)
```

Comment:
Not from this PR, but sharing in any case, `refetch-interval` is defined as a state, but no part in the code relies on it as a reactive state, so it could be a ref.